app/presenters/queries/expanded_link_set.rb

```diff
@@ -52,7 +52,7 @@ def expand_links(links)
         links.map do |link|
           item = all_web_content_items[link["target_content_id"]]
           next unless item
-          next_level = expand_level(link["target_content_id"]) if link['cycle'] == 'f'
+          next_level = expand_level(link["target_content_id"]) unless link['cycle']
```

Comment:
I am surprised this works, since link['cycle'] returns 't' or 'f'. Or is this a change in Rails 5 where it would now return a boolean?